### PR TITLE
protection 1

### PR DESCRIPTION
**Pull request by danggro**

### CODE REVIEW

**Comment by danggro** [comment]
okay thats good

**Review by danggro — COMMENTED** [review]
sip mantap

File: .github/workflows/deploy_render.yml
Comment (danggro) [inline .github/workflows/deploy_render.yml:1]: sip mantap

File: app.js
Comment (danggro) [inline app.js:1]: gile bro

File: .github/workflows/deploy_render.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Bump version and push tag
-        uses: anothrNick/github-tag-action@1.64.0
+        uses: anothrNick/github-tag-action@a2c70ae13a881faf2b4953baaa9e49731997ab36
```
Comment (danggro) [inline .github/workflows/deploy_render.yml:61]: :)

**Review by danggro — COMMENTED** [review]
:+1:

File: .github/workflows/deploy_render.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Bump version and push tag
-        uses: anothrNick/github-tag-action@1.64.0
```
Comment (danggro) [inline .github/workflows/deploy_render.yml:61]: aabbcc

Comment (danggro) [reply .github/workflows/deploy_render.yml:61]: asdas

File: app.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const PORT = process.env.PORT || 5000
 app.use(express.static('dist'))
 
 app.get('/version', (req, res) => {
-  res.send('pass versioning 15')
+  res.send('protection 1')
```
Comment (danggro) [inline app.js:10]: 112233

File: app.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ const PORT = process.env.PORT || 5000
 app.use(express.static('dist'))
 
 app.get('/version', (req, res) => {
-  res.send('pass versioning 15')
```
Comment (danggro) [inline app.js:10]: 2233111

File: .github/workflows/deploy_render.yml (hunk 2 above)
Comment (danggroo) [inline .github/workflows/deploy_render.yml:61]: asd

**Review by danggroo — APPROVED** [review]

File: .github/workflows/deploy_render.yml (hunk 1 above)
Comment (danggroo) [inline .github/workflows/deploy_render.yml:61]: 112233

File: app.js (hunk 3 above)
Comment (danggroo) [inline app.js:10]: 332211

**Review by danggroo — APPROVED** [review]

File: .github/workflows/deploy_render.yml (hunk 1 above)
Comment (danggroo) [reply .github/workflows/deploy_render.yml:61]: qwewer

File: app.js (hunk 3 above)
Comment (danggroo) [reply app.js:10]: werqwe

**Review by danggroo — APPROVED** [review]
sadqwe

File: .github/workflows/deploy_render.yml
Comment (danggroo) [inline .github/workflows/deploy_render.yml:1]: asdwqesad

File: app.js
Comment (danggroo) [inline app.js:1]: sadzxcads

**Review by danggroo — APPROVED** [review]

**Pull request merged**